src/components/Verification/ConnectionList.tsx

```diff
@@ -223,7 +301,7 @@ const ConnectionList = (props: {
 				onInputChange={searchInputChange}
 				refresh={refreshPage}
 				header={verification_header}
-				data={connectionList}
+				data={tableData}
```

Comment:
Which data table has? rename variable for the same.

Reply:
Yes.